src/instanceMethods/_main.js

```diff
@@ -0,0 +1,548 @@
+import {showWarningsForParams} from '../utils/params'
+import * as dom from '../utils/dom/index'
+import { swalClasses } from '../utils/classes'
+import { formatInputOptions, error, callIfFunction } from '../utils/utils'
+import setParameters from '../utils/setParameters'
+import globalState from '../globalState'
+import { openPopup } from '../utils/openPopup'
+
+export function _main (userParams) {
+  const ctor = this.constructor // dynamic reference to "the" (not necessarily our) constructor function
```

Comment:
`ctor` is quite a unique variable name, `constructor` would be better IMO

Reply:
I defined `ctor` just to make the reference shorter, it's not really needed. If you want the ref to be `constructor` then we might as well just make it `this.constructor` which is probably better than either. Sound ok?

Reply:
Yup :+1:

Reply:
Done in https://github.com/sweetalert2/sweetalert2/pull/1036/commits/1a4928986a27f73e5557684ca64f650bca7661e0